alcs-frontend/src/app/shared/mention-textarea/mention-textarea.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, ElementRef, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { CommentDto, CreateCommentDto, MentionDto, UpdateCommentDto } from '../../services/comment/comment.dto';
+import { UserService } from '../../services/user/user.service';
+
+@Component({
+  selector: 'app-mention-textarea',
+  templateUrl: './mention-textarea.component.html',
+  styleUrls: ['./mention-textarea.component.scss'],
+})
+export class MentionTextareaComponent implements OnInit {
+  @Input() fileNumber!: string;
+  @Input() isSaving = false;
```

Comment:
I don't see you pass isSaving anywhere, so many you just need to remove Input annotation?